src/DashboardService.Validation/Board/EditBoardRequestValidator.cs

```diff
@@ -2,12 +2,79 @@
 using LT.DigitalOffice.DashboardService.Data.Interfaces;
 using LT.DigitalOffice.DashboardService.Models.Dto.Requests.Board;
 using LT.DigitalOffice.DashboardService.Validation.Board.Interfaces;
+using LT.DigitalOffice.Kernel.Validators;
+using Microsoft.AspNetCore.JsonPatch;
+using Microsoft.AspNetCore.JsonPatch.Operations;
+using System;
+using System.Collections.Generic;
+using System.Threading;
 
 namespace LT.DigitalOffice.DashboardService.Validation.Board;
 
-public class EditBoardRequestValidator : AbstractValidator<PatchBoardRequest>, IEditBoardRequestValidator
+public class EditBoardRequestValidator : ExtendedEditRequestValidator<Guid, PatchBoardRequest>, IEditBoardRequestValidator
 {
-  public EditBoardRequestValidator(IBoardRepository boardRepository)
+  private readonly IBoardRepository _boardRepository;
+
+  public EditBoardRequestValidator(IBoardRepository repository)
+  {
+    _boardRepository = repository;
+
+    RuleFor(x => x)
+      .CustomAsync(async (x, context, ct) =>
+      {
+        foreach (var op in x.Item2.Operations)
+        {
+          await HandleInternalPropertyValidationAsync(op, x.Item1, context, ct);
+        }
+      });
+  }
+
+  private async System.Threading.Tasks.Task HandleInternalPropertyValidationAsync(
```

Comment:
Can you use only Task instead of "System.Threading.Tasks.Task"?

Reply:
VS gives an error